optimize/client/src/style.scss

```diff
@@ -7,12 +7,16 @@
 @use '@carbon/react/scss/fonts/sans';
 @use '@carbon/react/scss/themes';
 @use '@carbon/react/scss/theme';
+@use '@carbon/react/scss/type';
+
+// temporary solution until we have a global reset (this is a workaround to use carbon font with reset disabled)
+@include type.reset();
```

Comment:
We reenabled the reset, so do we still need this?

Reply:
yes, this loads the font from carbon library so we can remove the font file from `public`